### PR TITLE
Updated code-quality checks due to compute time & added binary-checks on other scripts

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a pull request to the WrongSecrets app!

### CODE REVIEW

File: aws/k8s-vault-aws-start.sh [hunk 1]
```diff
@@ -3,6 +3,20 @@
...
+    if ! [ -x "$(command -v "$var")" ]; then
+      echo "🔥 ${var} is not installed." >&2
+      exit 1
+    else
+      echo "🏄 $var is installed..."
+    fi
+  done
+}
+
+checkCommandsAvailable helm minikube jq vault sed grep docker grep cat
```
Comment (bendehaan) [inline aws/k8s-vault-aws-start.sh:18]:
```suggestion
checkCommandsAvailable helm minikube jq vault sed grep docker grep cat aws
```

File: gcp/k8s-vault-gcp-start.sh [hunk 2]
```diff
@@ -3,6 +3,20 @@
...
+    if ! [ -x "$(command -v "$var")" ]; then
+      echo "🔥 ${var} is not installed." >&2
+      exit 1
+    else
+      echo "🏄 $var is installed..."
+    fi
+  done
+}
+
+checkCommandsAvailable helm minikube jq vault sed grep docker grep cat
```
Comment (bendehaan) [inline gcp/k8s-vault-gcp-start.sh:18]:
```suggestion
checkCommandsAvailable helm minikube jq vault sed grep docker grep cat gcloud
```

**Review by bendehaan — CHANGES_REQUESTED** [review]
Is the vault binary actually necessary?

File: aws/k8s-vault-aws-start.sh (hunk 1 above)
Comment (nbaars) [reply aws/k8s-vault-aws-start.sh:18]: Vault is necessary, the script uses:

`vault kv put secret/secret-challenge`

**Comment by commjoen** [comment]
> Is the vault binary actually necessary?

yes ;( as we use vault to put values in... we could replace it with curl in the future, but rather just go with the flow now.

**Review by bendehaan — APPROVED** [review]

**Pull request merged**